src/crustls.h

```diff
@@ -6,6 +6,12 @@
 #include <stdint.h>
 #include <stdlib.h>
 
+/**
+ * The number of cipher suites supported by rustls. You may call `rustls_all_ciphersuites_get()`
+ * with indexes lower than this number.
+ */
+#define RUSTLS_ALL_CIPHERSUITES_LEN 9
```

Comment:
If you deploy a new version of crustls/rustls on a system as shared library, the application will not be recompiled and use the previous number of ciphers.